.gitlab-ci.yml

```diff
@@ -12,6 +12,14 @@ variables:
 .setup_rust: &setup_rust
     rustup default nightly
 
+.setup_tendermint: &setup_tendermint
+    curl -O https://raw.githubusercontent.com/tendermint/tendermint/develop/scripts/install/install_tendermint_ubuntu.sh &&
+    chmod +x install_tendermint_ubuntu.sh &&
+    ./install_tendermint_ubuntu.sh &&
+    source ~/.profile &&
+    tendermint init &&
+    sed -i '/^laddr = "tcp:\/\/0.0.0.0:26657"/laddr = "tcp:\/\/0.0.0.0:3030"/' ~/.tendermint/config/config.toml
```

Comment:
`laddr` is overridable through command line arguments when `tendermint node` is run. No need to sed the file.

Reply:
I cannot find how to override that

Reply:
`tendermint node --help` returns:
`--p2p.laddr string`

Reply:
yeah I also found it

Reply:
Oh, I see you are overriding RPC laddr. Nevermind then.

Reply:
https://tendermint.com/rpc/#configuration